staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -635,11 +617,13 @@ func (c *Cacher) listItems(listRV uint64, key string, pred storage.SelectionPred
 	return c.watchCache.WaitUntilFreshAndList(listRV, pred.MatcherIndex(), trace)
 }
 
-func (c *Cacher) list(ctx context.Context, key string, opts storage.ListOptions, listObj runtime.Object, recursive bool) error {
+// List implements storage.Interface.
+func (c *Cacher) List(ctx context.Context, key string, opts storage.ListOptions, listObj runtime.Object) error {
```

Comment:
this change means that existing callers of `List(ctx context.Context, key string, opts storage.ListOptions, listObj runtime.Object)` that don't know about the new Recursive field when they construct their `ListOptions` struct will break, right?